src/main/app/src/components/Production.js

```diff
@@ -14,48 +15,84 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const getMaterials = async () => {
+  const api = '/api/materials/';
+  const got = await fetch(api);
+  const json = await got.json();
+
+  if (!json.materials) return [];
+
+  return json.materials.map(materials => {
+    materials.allMaterials = {
+      onClick: () => alert(`Test: ${materials.material}`),
+    };
+    return materials;
+  });
+};
+
+const getProducts = async () => {
+  const api = '/api/products/';
+  const got = await fetch(api);
+  const json = await got.json();
+
+  if (!json.products) return [];
+
+  return json.products.map(products => {
+    products.allProducts = {
+      onClick: () => alert(`Test: ${products.product}`),
```

Comment:
Same here with this alert, easiest would be to just replace with no alert for now: `onClick: () => {},`